wnfs/src/private/node/serializable.rs

```diff
@@ -52,12 +52,11 @@ pub(crate) struct PrivateNodeHeaderSerializable {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub(crate) struct PrivateRefSerializable {
-    #[serde(serialize_with = "crate::utils::serialize_byte_slice32")]
-    #[serde(deserialize_with = "crate::utils::deserialize_byte_slice32")]
+    #[serde(with = "serde_byte_array")]
     pub label: HashOutput,
     #[serde(rename = "snapshotKey")]
     pub snapshot_key: SnapshotKey,
-    #[serde(rename = "temporalKey")]
```

Comment:
Accidental rename removal :sweat_smile:

Reply:
May be worth just switching to that one struct-level directive that renames everything into camel-case. But we can do that in another PR, this way we could make sure that it doesn't actually change snapshots ;)